src/main/scala/ciris/aws/ssm/package.scala

```diff
@@ -1,27 +1,24 @@
 package ciris.aws
 
-import cats.effect.{Blocker, IO, Resource}
+import cats.effect.kernel.{Resource, Sync}
 import ciris.ConfigValue
 import software.amazon.awssdk.auth.credentials.{AwsCredentialsProvider, DefaultCredentialsProvider}
 import software.amazon.awssdk.regions.Region
 import software.amazon.awssdk.services.ssm.SsmClient
 
 package object ssm {
-  def params(
-    blocker: Blocker,
-    region: Region
-  ): ConfigValue[Param] =
-    params(blocker, region, DefaultCredentialsProvider.create())
+  def params[F[_]: Sync](region: Region
+  ): ConfigValue[F, Param[F]] =
+    params(region, DefaultCredentialsProvider.create())
 
-  def params(
-    blocker: Blocker,
+  def params[F[_]: Sync](
     region: Region,
     credentials: AwsCredentialsProvider
-  ): ConfigValue[Param] =
+  ): ConfigValue[F, Param[F]] =
     ConfigValue.resource {
       Resource
-        .fromAutoCloseable {
-          IO {
+        .fromAutoCloseable[F, SsmClient] {
+          Sync[F].pure {
```

Comment:
Are we sure this is pure? wasn't using an `IO.pure` previously...

Reply:
Yeah good catch. I changed `.pure` to `.delay`.

Reply:
I agree. Probably need to use Sync[F].delay instead

Reply:
Just saw your change, makes sense now